tests/integration/tier0/non-destructive/kernel/test_kernel_check_verification.py

```diff
@@ -181,6 +177,12 @@ def kernel(shell):
         # Set the latest kernel as the one we want to reboot to
         shell("grub2-set-default '{}'".format(full_name.strip()))
 
+        # Remove the mocked repofile
+        if "alma-8" in SYSTEM_RELEASE_ENV:
+            shell(f"rm -f /etc/yum.repos.d/alma_old.repo")
```

Comment:
Maybe you could use the `moc_repo` variable here as well:
```rm -f /etc/yum.repos.d/{mock_repo}.repo```

Reply:
Not working, it fails as the variable is out of reach